package.json

```diff
@@ -53,7 +53,10 @@
     "db:generate": "npx drizzle-kit generate",
     "jsonSchema:generate": "npx -y tsx scripts/generateJsonSchema.ts > site/static/config-schema.json",
     "audit:fix": "npm audit fix && npm audit fix --prefix src/web/nextui && npm audit fix --prefix site",
-    "prepublishOnly": "npm run build:clean && npm run build"
+    "prepublishOnly": "npm run build:clean && npm run build",
+    "dev:server": "nodemon src/server/index.ts",
+    "dev:web": "npm run dev --prefix src/web/nextui",
+    "dev": "concurrently \"npm run dev:server\" \"npm run dev:web\""
```

Comment:
nit, document these somewhere. This is fo web development but it's not obvious at first glance.

Reply:
added these to `contributing.md` https://github.com/promptfoo/promptfoo/pull/1599/files#diff-e6c8dab39e2dcea6882ed534980280aab6cd0598033b8291d1da771e7ce9fde7

https://www.promptfoo.dev/docs/contributing/